frontend/src/modules/member/pages/member-list-page.vue

```diff
@@ -119,6 +120,41 @@ export default {
 
     await this.doFetchIntegrations()
     await this.doFetchCustomAttributes()
+
+    const { joinedFrom, activeFrom } = this.$route.query
+    if (joinedFrom) {
+      await this.updateFilterAttribute({
+        custom: false,
+        defaultOperator: 'gt',
+        defaultValue: joinedFrom,
+        expanded: false,
+        label: 'Joined date',
+        name: 'joinedAt',
+        operator: 'gt',
+        type: 'date',
+        value: joinedFrom
```

Comment:
Should we validate the value type? To see if it matches what we want? Right now if I write `joinedFrom=string`, the application breaks